src/elm/Pages/MyAccount/View.elm

```diff
@@ -20,12 +20,28 @@ view user =
 
                 _ ->
                     ""
+        login =
+            case user of
+                Success user' ->
+                    user'.login
+
+                _ ->
+                    ""
+        avatar =
+            case user of
+                Success user' ->
+                    img [ src user'.avatarUrl ]
+                        []
+
+                _ ->
```

Comment:
if there is no user, then you shouldn't have access to my account. So I would change it to be `_ -> div [] []`, as that icon will never appear anyway